telegram/mod.ts

```diff
@@ -1,130 +1,25 @@
-import {
-  json,
-  PathParams,
-  serve,
-  validateRequest,
-} from "https://deno.land/x/sift@0.4.2/mod.ts";
-
-// For all requests to "/<TOKEN>" endpoint, we want to invoke handleTelegram() handler.
-// Recommend using a secret path in the URL, e.g. https://www.example.com/<token>.
-serve({
-  "/": () => new Response("Welcome to the Telegram Bot site."),
-  "/:slug": handleTelegram,
-});
-
-// The main logic of the Telegram bot is defined in this function.
-async function handleTelegram(request: Request, params?: PathParams) {
-  // Gets the environment variable TOKEN
-  const TOKEN = Deno.env.get("TOKEN")!;
-  // Gets the environment variable BOT_NAME
-  const BOT_NAME = Deno.env.get("BOT_NAME")!;
-
-  // If the environment variable TOKEN is not found, throw an error.
-  if (!TOKEN) {
-    throw new Error("environment variable TOKEN is not set");
-  }
-
-  // For using a secret path in the URL, e.g. https://www.example.com/<token>. If wrong url return "invalid request".
-  if (params?.slug != TOKEN) {
-    return json(
-      { error: "invalid request" },
-      {
-        status: 401,
-      },
-    );
-  }
-
-  // Make sure the request is a POST request.
-  const { error } = await validateRequest(request, {
-    POST: {},
-  });
-
-  // validateRequest populates the error if the request doesn't meet
-  // the schema we defined.
-  if (error) {
-    return json({ error: error.message }, { status: error.status });
-  }
-
-  // Get the body of the request
-  const body = await request.text();
-  // Parse the raw JSON body from Telegrams webhook.
-  const data = await JSON.parse(body);
-
-  // Check if the method is a POST request and that there was somthing in the body.
-  if (request.method === "POST") {
-    // Cheack if the command was "/ping".
-    if (
-      data && data["message"] && data["message"]["text"] &&
-      (data["message"]["text"].toLowerCase() == "/ping" ||
-        data["message"]["text"].toLowerCase() ==
-          "/ping@" + BOT_NAME.toLowerCase())
-    ) {
-      // Store the chat id of the Group Chat, Channel or PM.
-      const chatId: number = data["message"]["chat"]["id"];
-
-      // Calls the API service to Telegram for sending a message.
-      const { dataTelegram, errors } = await sendMessage(
-        chatId,
-        "Pong",
-        TOKEN,
-      );
-
-      if (errors) {
-        console.error(errors.map((error) => error.message).join("\n"));
-        return json({ error: "couldn't create the message" }, {
-          status: 500,
-        });
+import { serve } from "https://deno.land/std@0.149.0/http/mod.ts";
+import { webhookCallback } from "https://deno.land/x/grammy@v1.9.2/mod.ts";
+import { bot } from "./bot.ts";
+
+await bot.init();
+
```

Comment:
This will be done automatically by the framework as soon as the first message arrives.
```suggestion
```